app/views/shared/review_application/_income_payments_and_assets_for_bank_statement_upload.html.erb

```diff
@@ -18,7 +22,7 @@
       <% else %>
         <%= render "shared/check_answers/employed_income", partner: false, read_only: true %>
       <% end %>
-    <% elsif @legal_aid_application.applicant_employed? %>
+    <% elsif @legal_aid_application.applicant.employed? %>
```

Comment:
The removed method `applicant_employed?` was using safe operator `&.`, I presume to avoid `NoMethodError`

Should this be applied if we want to be safe and not risk a `nil` error or is this no longer a concern?

Reply:
I assumed this method was a old method leftover from before we migrated a lot of methods (and data) away from the `LegalAidApplication` model into the specific `Partner` and `Applicant` models. This page was the only location where this method was called, hence I removed it and replaced with the applicant/partner methods. 

There are _many, many_ instances where we chain method calls on the applicant (or the partner) through the `@legal_aid_application` instance and do not use the safe operator! IMO it is highly unlikely that the applicant would not exist but perhaps this is something we should look at, so thanks for pointing it out.